### ISSUE TITLE
Shouldn't the more powerful AMD have more advantage over the integrated Intel chipset?

### ISSUE THREAD

**Issue by manuel-arno-korfmann-webentwicklung** [plaidml/plaidml#1036]
https://github.com/plaidml/plaidml/blob/f7dda910258ebf4cd9b1ab2093442d135800b0b8/networks/scitile/uw_toroidal_shell/docs/uw_toroidal_shell_doc.tex#L54

**Comment by Degerz** [comment]
Given that PlaidML is maintained by Intel employees and that AMD doesn't care about OpenCL this isn't a surprise ... 

You should just use the ROCm [fork](https://github.com/ROCmSoftwarePlatform/tensorflow-upstream) of Tensorflow which is actively developed by AMD or you can get the [upstream](http://ml-ci.amd.com:21096/job/tensorflow-rocm-nightly/) version [through](https://github.com/tensorflow/tensorflow) community supported builds. 

[There](https://hub.docker.com/r/rocm/pytorch/tags) are even docker images of PyTorch for ROCm.

**Comment by flaub** [comment]
This is an open source project, so we welcome anyone with patches for any target they'd like to improve performance on.
